Error: Cannot find module 'lodash/merge.js'

### Description of the bug/issue

When I updated Nightwatch to 3.1, I expected my tests to execute, but instead I received the error that 'lodash/merge.js' is missing. 



### Steps to reproduce

1. Modify `package.json` to include `"nightwatch": "^3.1.0"`
2. `npx npkill`
3. `pnpm install`
4. `npx nightwatch ./test`
5. See error


### Sample test

_No response_

### Command to run

_No response_

### Verbose Output

```fundamental
/opt/homebrew/bin/pnpm run testCreateOrder

> <redacted>-nightwatch@1.0.0 testCreateOrder /Users/<redacted>
> npx nightwatch ./test/createOrder.ts

node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module 'lodash/merge.js'
Require stack:
- /Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/dist/core/client.js
- /Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/dist/index.js
- /Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/bin/runner.js
- /Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/bin/nightwatch
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (/Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/dist/core/client.js:4:21)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/dist/core/client.js',
    '/Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/dist/index.js',
  '/Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/bin/runner.js',
    '/Users/<redacted>/node_modules/.pnpm/nightwatch@3.1.0/node_modules/nightwatch/bin/nightwatch'
  ]
}

Node.js v18.16.1
 ELIFECYCLE  Command failed with exit code 1.

Process finished with exit code 1
```


### Nightwatch Configuration

_No response_

### Nightwatch.js Version

3.1.0

### Node Version

v18.16.1

### Browser

n/a

### Operating System

MacOS Ventura 13.4.1

### Additional Information

Apple Silicon (M2) CPU

In the `package.json` of Nightwatch, I can see that `lodash` is only a devDependency. That might be the issue.

--- a/lib/core/client.js
+++ b/lib/core/client.js
@@ -1,7 +1,7 @@
 const EventEmitter = require('events');
 const {Key, Capabilities, Browser} = require('selenium-webdriver');
 
-const lodashMerge = require('lodash/merge.js');
+const lodashMerge = require('lodash.merge');
 const HttpOptions = require('../http/options.js');
 const HttpRequest = require('../http/request.js');
 const Utils = require('../utils');

